### ISSUE TITLE
Gather initial list of OER tools

### ISSUE THREAD

**Issue by literarymachine** [hbz/oerworldmap#1249]
For #170 an initial list of tools used for the [services on the OER World Map](https://oerworldmap.org/resource.csv?q=&filter.about.%40type=Service) should be gathered by contacting the providers by email and asking for the technical basis of their platform.

**Comment by literarymachine** [comment]
@karindr you could start with the following list of services that have explicitly specified an email address (which is now also included in the CSV export): https://oerworldmap.org/resource/?q=_exists_%3Aabout.email&filter.about.%40type=Service

**Comment by karindr** [comment]
I work on the email now.

Is it possible to change the query above to the opposite? I realised that less than 50 % of the services are listed with email address. If I have a list of services without email I could have a look if some important services are missing. Then I can look for some more email adresses.

**Comment by literarymachine** [comment]
> Is it possible to change the query above to the opposite?

Sure! https://oerworldmap.org/resource/?q=_missing_:about.email&filter.about.@type=Service

> I realised that less than 50 % of the services are listed with email address.

I think often times the provider has an email address listed, but this is currently not present in the default export list. The quickest way to get at these is this [SPARQL Query](https://oerworldmap.org/sparql/query?q=PREFIX+%3A+%3Chttp%3A%2F%2Fschema.org%2F%3E%0D%0A%0D%0ASELECT+DISTINCT+%3Fservice+%3FproviderEmail+WHERE+%7B%0D%0A++%3Fservice+a+%3AService+.%0D%0A++%3Fprovider+%3Aprovides+%3Fservice+.%0D%0A++%3Fprovider+%3Aemail+%3FproviderEmail+.%0D%0A++FILTER+NOT+EXISTS+%7B+%3Fservice+%3Aemail+%3FserviceEmail+%7D%0D%0A%7D). It lists all services that don't have an email of their own with their provider's email. Would that work for now?

**Comment by karindr** [comment]
I sent out an English and a German mail to these adsresses and started to collect the answers in a google table. Some answers are from twitter too.
See:
https://docs.google.com/spreadsheets/d/1lLD5XcrRDHbtXXLUahNhsHg8-5tEJ6mfab7AdMm70g8/edit?usp=sharing
I will continue till I am on vacation. 
First results: Not so much answers/services but yet a long list of tools.

Maybe we should think of a dropdown list to choose from for services with the possibility to add new tools.

**Comment by literarymachine** [comment]
> First results: Not so much answers/services but yet a long list of tools.

I find this list quite interesting!

> Maybe we should think of a dropdown list to choose from for services with the possibility to add new tools.

You mean to link from a service to a tool, similar as the drop down for licenses? That is the idea!

**Comment by karindr** [comment]
Yes, like for licences or subjects. But you must be able to add new tools, too.

**Comment by literarymachine** [comment]
> Yes, like for licences or subjects. But you must be able to add new tools, too.

Being able to dynamically add to the list would require some new development. The plan was to gather an initial list and then create a controlled vocabulary (like for licences and subjects) first. At the workshop in Weimar we want to add Tools to our data model, so that they can have their own entries on the map. Then they can be linked to like an Organization as a provider of a Service.

**Comment by karindr** [comment]
I'm not shure if tools are so importent for the world map to give them an own datatype. As you can see in the list there are as little tools as irfantview or services like twitter. These tools are in my opinion not strongly enough related to OER to collect them as own type on the map. 

I like the idea of tools as additional information to other datatypes (like subjects for services and events). Technically maybe as an extra field for tools on the service template working like the tags field: You start entering the name and it suggests tools, that are allready on the map.

**Comment by karindr** [comment]
Let me add a comment from Thomas Haubner from tutory. He mentions the point that tools are somehow also included in services for example authoring tools. Here's what he wrote:
Also ich war immer unsicher, was Service meint. Da scheint sich ja auch eine Menge zu mischen. Vielleicht muss man diese Kategorie mal aufräumen und nochmal unterteilen denn eigentlich würden Tools darin gehören von der aktuellen Ordnung her. Wenn es eine neue Überkategorie Tools gibt, dann würde diese mit Service konkurrieren. Fällt mir schwer da eine Lösung zu empfehlen. Persönlich finde ich es natürlich spannend, die Tools schnell findbar zu haben.

**Comment by literarymachine** [comment]
> I'm not shure if tools are so importent for the world map to give them an own datatype.

You should really discuss this with @trugwaldsaenger then, because he said they are and this is the plan we have for the workshop. I would rather not like to implement something there that we don't need.

**Comment by literarymachine** [comment]
> Also ich war immer unsicher, was Service meint.

We have had this criticism multiple time already, and I can understand it. We have for some reason never reacted to it, though.

 > Wenn es eine neue Überkategorie Tools gibt, dann würde diese mit Service konkurrieren.

I don't think so. Just a totally random example: the [SUNY OER Repository](https://dspace.sunyconnect.suny.edu/handle/1951/65009) is what we currently call a Service (repositories etc), and it is implemented using [DSpace](http://www.dspace.org/), which is a software, a tool. A tool can be instantiated several times, a service is a running product people actually use, an instance of a tool or perhaps even using many tools. Maybe @acka47 can add some input from the data modeling point of view.

**Comment by karindr** [comment]
There some services for example tutory that are a kind of toot, as tag we use "authoring tool" for these services. That is where we must make clear what is a tool for us and what is a service. People aksed for tools they use, submit nearly everything. See the list from mail and twitter answers: 
https://docs.google.com/spreadsheets/d/1lLD5XcrRDHbtXXLUahNhsHg8-5tEJ6mfab7AdMm70g8/edit?usp=sharing
I also talked to @trugwaldsaenger about an own datatype for tools or not. He said that this datatype can be in the background of the system like the type funding. What role it should play on the website could be decides in another step.

**Comment by literarymachine** [comment]
ping @tomashkow

**Comment by thomashkov** [comment]
hey everyone,
we understand tutory as authoring tool as well as platform, because authored content can be published and also distributed (shared and embedded). So for our OER-Service we are basically using our own OER-tool, which is right now open for private users (e.g. teachers and educators).

I agree to the idea of assigning tools to services. Although you have to ask yourself, what users primarily look for. If they are looking for services more then people and organizations, maybe a higher category would be interesting.

I suggest that as a distinguishing criterion OER-tools may have integrated features for licensing and "license management". With that in mind your list would get much more comprehensive and usable.

**Comment by karindr** [comment]
Another point of view to tools/services: Christof Mauersberger from memucho sees his service also as tool, see: http://open-educational-resources.de/interview-lernplattform-memucho/

**Comment by trugwaldsaenger** [comment]
Here come some late comments on this issue:

**1. Distinction Tools Services**
I would consider tutory primarily as an service. The underlying tool (=software) could be a tool as well, but since it is not openly licensed (nor available for reuse by licensing) I would not add it as a tool, because only tutory can use it.

The general rule is: the concrete product (with a special name and a special URL, terms of conditions etc) is a service, the used software (e.g. DSpace for repositories, Moodle for LMS and Mediawiki for Wikis) are the tools. The idea is that we can see, which service uses which tools.

I also like how @literary machine put it:

> A tool can be instantiated several times, a service is a running product people actually use, an 
> instance of a tool or perhaps even using many tools. 

**2) Tag "authoring tool"**
I have to admit that I used the tag "authoring tool" for tutory. This was imprecise and might have been misleading. At that time I did not want to point out that tutory is a tool, but that tutory can be used to create documents, that it is with other words an editor.

**3) Subcategories for services**
> Also ich war immer unsicher, was Service meint. Da scheint sich ja auch eine Menge zu mischen. >Vielleicht muss man diese Kategorie mal aufräumen und nochmal unterteilen

Yup,that`s true, Services should be divided in subcategories. The big question is, what are the subcategories? I created the an own issue for this: 

4) Services, which fit into several subcategories:
Thomas pointed out that
we understand tutory as authoring tool as well as platform, because authored content can be published and also distributed (shared and embedded).

This is a correct observation. Many services fit to multiple subcategories as defined in xxx. I propose to continue the discussion there.

5) memucho consider themselves as "tools"
Words are deceptive. If you use more every day language, it is OK to speak of a service as a "tool". In our classification it is primarily a service. If the software is available for reuse, we could add it as a tool as well.
